_rules/iframe-identical-name-equivalent-purpose-4b1c6c.md

```diff
@@ -37,12 +42,13 @@ The `iframe` elements in each set of target elements embed the the [same resourc
 
 ## Accessibility Support
 
-_There are no major accessibility support issues known for this rule._
+This rule assumes that assistive technologies are exposing all `iframe` elements on the page in the same way no matter which [document tree](https://dom.spec.whatwg.org/#document-trees) they are in. If an assistive technology requires the user to "enter" an `iframe` or a [shadow tree][] before exposing its content (notably nested `iframe`), then it is possible for two `iframe` to have identical name but embed different resources without breaking [Success Criterion 4.1.2: Name, Role, Value][sc412] (if said `iframe` are in separate [documents][document] or [shadow trees][shadow tree])
```

Comment:
Editorial: You can't "break" a success criterion. An SC can be either satisfied or not satisfied.